### PR TITLE
adding in the ability to handle the telephoneNumber format and the em…

### PR DESCRIPTION
**Pull request by DanielConnelly**
…ail address format

### CODE REVIEW

File: app/uk/gov/hmrc/gform/core/parsers/FormatParser.scala [hunk 1]
```diff
@@ -21,7 +21,7 @@ import uk.gov.hmrc.gform.core.Opt
 import uk.gov.hmrc.gform.sharedmodel._
 import BasicParsers._
 import uk.gov.hmrc.gform.formtemplate._
-import uk.gov.hmrc.gform.sharedmodel.formtemplate._
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.{ Email, TelephoneNumber, _ }
```
Comment (tenochdlc) [inline app/uk/gov/hmrc/gform/core/parsers/FormatParser.scala]: aren't Email and TelephoneNumber redundant with the "_ " there

**Comment by tenochdlc** [comment]
a couple of unit tests would be useful I think

**Comment by lutnos** [comment]
Yes, and the Jira ID in at least a commit message

**Comment by DanielConnelly** [comment]
@lutnos the Jira Id is the name of the branch.

**Comment by pawelpanasewicz** [comment]
LGTM

**Pull request merged**